File: first-exercises/c/second.c
#include <stdio.h>

int main() {
    float preco_mercadoria, percentual_desconto, desconto, preco_a_pagar;

    printf("Digite o preço da mercadoria: ");
    scanf("%f", &preco_mercadoria);

    printf("Digite o percentual de desconto: ");
    scanf("%f", &percentual_desconto);

    desconto = (percentual_desconto / 100) * preco_mercadoria;
    preco_a_pagar = preco_mercadoria - desconto;

    printf("O valor do desconto é R$%.2f\n", desconto);
    printf("O preço a pagar é R$%.2f\n", preco_a_pagar);

    return 0;
}

File: first-exercises/python/second.py
# Solicita o preço da mercadoria e o percentual de desconto
preco_mercadoria = float(input("Digite o preço da mercadoria: "))
percentual_desconto = float(input("Digite o percentual de desconto: "))

# Calcula o valor do desconto
desconto = (percentual_desconto / 100) * preco_mercadoria

# Calcula o preço a pagar
preco_a_pagar = preco_mercadoria - desconto

# Exibe o valor do desconto e o preço a pagar
print(f"O valor do desconto é R${desconto:.2f}")
print(f"O preço a pagar é R${preco_a_pagar:.2f}")
